docs/src/modules/components/AppFrame.js

```diff
@@ -229,16 +226,14 @@ function AppFrame(props) {
               onClick={handleLanguageIconClick}
               data-ga-event-category="AppBar"
               data-ga-event-action="language"
+              startIcon={<LanguageIcon fontSize="small" />}
+              endIcon={<ExpandMoreIcon fontSize="small" />}
             >
-              <LanguageIcon />
               <Hidden xsDown implementation="css">
```

Comment:
This div creates an invalid DOM structure: https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-17600--material-ui.netlify.com%2F

Reply:
We might want to trigger the small version for a larger screen width, it flex shrinks otherwise:

![Capture d’écran 2019-09-29 à 18 43 44](https://user-images.githubusercontent.com/3165635/65835641-25be5a00-e2e9-11e9-98f8-a99383d7643d.png)

Reply:
Hidden doesn't support the `component` prop. Do we go with the default `js` implementation or fix that separately?